pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -0,0 +1,662 @@
+								},
+								Limits: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse("1"),
+									corev1.ResourceMemory: resource.MustParse("1Gi"),
+								},
+							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      DeploymentName,
+									MountPath: "/srv/secrets/vpn-server",
+								},
+								{
+									Name:      "vpn-seed-server-tlsauth",
+									MountPath: "/srv/secrets/tlsauth",
+								},
+								{
+									Name:      "vpn-seed-server-dh",
+									MountPath: "/srv/secrets/dh",
+								},
+							},
+						},
+						{
+							Name:            "envoy-proxy",
+							Image:           v.imageAPIServerProxy,
+							ImagePullPolicy: corev1.PullIfNotPresent,
+							SecurityContext: &corev1.SecurityContext{
+								Capabilities: &corev1.Capabilities{
+									Add: []corev1.Capability{
+										"NET_BIND_SERVICE",
+									},
+								},
+							},
+							Command: []string{
+								"envoy",
+								"--concurrency",
+								"2",
+								"-c",
+								"/etc/envoy/envoy.yaml",
+							},
+							Resources: corev1.ResourceRequirements{
+								Requests: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse("20m"),
+									corev1.ResourceMemory: resource.MustParse("20Mi"),
+								},
+								Limits: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse("200m"),
+									corev1.ResourceMemory: resource.MustParse("300Mi"),
+								},
+							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "envoy-config",
+									MountPath: "/etc/envoy",
+								},
+								{
+									Name:      DeploymentName,
+									MountPath: "/etc/tls",
+								},
+							},
+						},
+					},
+					TerminationGracePeriodSeconds: pointer.Int64Ptr(30),
+					Volumes: []corev1.Volume{
+						{
+							Name: DeploymentName,
+							VolumeSource: corev1.VolumeSource{
+								Secret: &corev1.SecretVolumeSource{
+									SecretName: DeploymentName,
+								},
+							},
+						},
+						{
+							Name: "vpn-seed-server-tlsauth",
+							VolumeSource: corev1.VolumeSource{
+								Secret: &corev1.SecretVolumeSource{
+									SecretName: "vpn-seed-server-tlsauth",
+								},
+							},
+						},
+						{
+							Name: "vpn-seed-server-dh",
+							VolumeSource: corev1.VolumeSource{
+								Secret: &corev1.SecretVolumeSource{
+									SecretName: "vpn-seed-server-dh",
+								},
+							},
+						},
+						{
+							Name: "envoy-config",
+							VolumeSource: corev1.VolumeSource{
+								ConfigMap: &corev1.ConfigMapVolumeSource{
+									LocalObjectReference: corev1.LocalObjectReference{
+										Name: "vpn-seed-server-envoy-config",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+		if v.nodeNetwork != nil {
+			deployment.Spec.Template.Spec.Containers[0].Env = append(deployment.Spec.Template.Spec.Containers[0].Env, corev1.EnvVar{Name: "NODE_NETWORK", Value: *v.nodeNetwork})
+		}
```

Comment:
```suggestion

		if v.nodeNetwork != nil {
			deployment.Spec.Template.Spec.Containers[0].Env = append(deployment.Spec.Template.Spec.Containers[0].Env, corev1.EnvVar{Name: "NODE_NETWORK", Value: *v.nodeNetwork})
		}

```

readability

Reply:
Maybe I am blind due to the sun, but what is the difference in the above statements? Just the additional empty lines before and after the if block?

Reply:
Yeah, please consider it minor, I just suggested it because I easily overlooked this because everything was very much squeezed together.